configs/mx7ulpea-ucom_lmp_defconfig

```diff
@@ -80,3 +80,5 @@ CONFIG_OPTEE_LOAD_ADDR=0x60800000
 CONFIG_OPTEE_TZDRAM_SIZE=0x02000000
 CONFIG_OPTEE_TZDRAM_BASE=0x9e000000
 CONFIG_BOOTM_OPTEE=y
+CONFIG_TEE=y
+CONFIG_CMD_SMC=y
```

Comment:
Can you run 'make mx7ulpea-ucom_lmp_defconfig && make savedefconfig && cp defconfig configs/mx7ulpea-ucom_lmp_defconfig'?

That way the configs will be added at the right order by u-boot itself.

Reply:
ok